free-2.md

```diff
@@ -1,44 +1,51 @@
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+This idea helps eliminate ambiguity in laws. This language would help legislators in both developing laws and referencing laws. With a computer based language, it would be significantly easier to look up laws. For instance, there may be several laws that come up when specific key words are searched(e.g. "shoplifting with weapon law") that differ drastically based on context (state-state level, scale of the shope, state-national level). Additionally, legislators could avoid drafting a similar law that was previously repealled or still enact. As of now, it seems like drafting a law requires a significant amount of man power just to reference check thoroughly. 
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL is appropriate because it is deliberate and strict. Unlike English, this DSL will allow little room for ambiguity as it will have fields such as who/what this law is intended for, when this law will be enacted, who/what this law will effect. This DSL addresses a longstanding need for clarity and conciseness in law making.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I am currently taking a public policy course off campus and one thing made extremely obvious is how convoluted the legislation process has grown to be. You don't have to be taking the course to understand just how slow the process can be. Although it can be argued ambiguity is a virtue (which is something we discussed in class e.g. the Constition is relevant today only because of its ambiguity), I think there is significant room for optimization.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Law making without B. S.
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+A legislator may interact with the lanuage by entering statements line by line, with each statement being some rule that adheres to the syntax of the language such that it can be made some logical predicate. Programming would ideally look closer to natural language with a few syntactical oddities (maybe logic symbols) to ensure clarity and enforce rules/logic. I would argue this is the right way to interact with laws because legislators can focus more on the content and less about the semantics.  
 
```

Comment:
In theory, this makes a lot of sense. In practice, it seems incredibly complicated and with lots of room for ambiguity. The language would need to account for so many different human actions and anticipate the format of future laws. It might be difficult to make it intuitive for the legislator to use, especially where readability is concerned.